entity-service/src/test/kotlin/com/egm/stellio/entity/authorization/Neo4jAuthorizationServiceTest.kt

```diff
@@ -298,26 +299,29 @@ class Neo4jAuthorizationServiceTest {
         assertTrue {
             countAndAuthorizedEntities.second.all {
                 it.type == "Beekeeper" && it.properties.containsKey(AUTH_PROP_RIGHT)
-            }
-        }
-        assertTrue(countAndAuthorizedEntities.second.any { it.properties.containsKey(AUTH_PROP_SAP) })
-        assertTrue {
-            countAndAuthorizedEntities.second.any {
-                it.properties[AUTH_PROP_RIGHT] == mutableMapOf(
-                    JsonLdUtils.JSONLD_TYPE to JsonLdUtils.NGSILD_PROPERTY_TYPE.uri,
-                    JsonLdUtils.NGSILD_PROPERTY_VALUE to mapOf(
-                        JsonLdUtils.JSONLD_VALUE_KW to AccessRight.R_CAN_ADMIN.attributeName
-                    )
-                ) &&
-                    it.properties.containsKey(AccessRight.R_CAN_READ.attributeName) &&
-                    it.properties[AccessRight.R_CAN_READ.attributeName] == users.map {
-                    mutableMapOf(
-                        JsonLdUtils.JSONLD_TYPE to JsonLdUtils.NGSILD_RELATIONSHIP_TYPE.uri,
-                        JsonLdUtils.NGSILD_RELATIONSHIP_HAS_OBJECT to mapOf(
-                            JsonLdUtils.JSONLD_ID to it
-                        )
+            } &&
```

Comment:
if one condition fails in this HUGE assert, it will be a pain to find the one that failed. better separate into distinct asserts

Reply:
Ok I do this